lib/AdobeMarketingCloud/HttpClient.php

```diff
@@ -206,12 +206,16 @@ public function getLastResponse()
      *
      * @return  array   the response
      */
-    protected function decodeResponse($response)
+    protected function decodeResponse($response, $options = array())
     {
-        switch ($this->options['format'])
+        if (count($options) <= 0) {
```

Comment:
I don't believe that `count()` can return a value that is < 0 so this may be more readable if you use `==`.